### DISCUSSION TITLE
Include current program feature

### DISCUSSION THREAD

**Discussion by jtaudien** [pybricks/support#886] [Q&A] [answered]
I saw in a video by Jean Galea that there used to be a feature called "Include current program" on the code.pybricks.com website to store the current program in non-volatile memory. It appears that the feature is no longer available. I bought the off-road buggy for my kids for Christmas and was hoping to get help with keeping the current program on the Powered UP hub. Thanks@

**Comment by dlech** [comment, marked as answer]
The current program is now saved to non-volatile memory when you power off the hub, so this switch in Pybricks Code is no longer needed. Just run the program once with Pybricks Code to get it on the hub, then you can start the program with the green button on the hub. If you turn the hub off and on again, the program is still on the hub. If you pull the batteries without turning off power with the button first, any new program won't be saved.

**Reply by jtaudien** [reply]
Thank you so much for your quick reply! It works as you described and my kids are having a blast with their car.

**Reply by vortexico** [reply]
In that case: could the documentation be updated?
At least here:
https://pybricks.com/install/technic-boost-city/#saving-a-program-on-the-hub
is stated that this switch is needed to save.

thank you in advance

**Reply by dlech** [reply]
Thanks for catching that. I made an issue for it to remind us to do that.
